awx/ui/src/screens/Job/JobDetail/JobDetail.js

```diff
@@ -361,6 +361,14 @@ function JobDetail({ job, inventorySourceLabels }) {
             value={t`True`}
           />
         )}
+        {typeof job.forks === 'number' &&
+          (
+            <Detail
+              dataCy="forks"
+              label={t`Forks`}
+              value={`${job.forks}`}
+            />
+        )}
```

Comment:
```suggestion
        {typeof job.forks === 'number' && (
          <Detail dataCy="forks" label={t`Forks`} value={`${job.forks}`} />
        )}
```

Reply:
looks like suggested edit made the format checking unhappy, sorry 😅 Let's see if this one is better

Reply:
We will also want to add a popover tooltip for this detail item I think. That can be done as a separate PR too.